src/interface.jl

```diff
@@ -40,7 +40,12 @@ mutable struct Interface
     function Interface()
         host = ENV["MOS_BACKEND_HOST"]
         port = ENV["MOS_BACKEND_PORT"]
-        url = "http://$host:$port/api/"
+        if port == 443
```

Comment:
I think we need port == "443" here (value of port will be or type string)

Reply:
Updated with latest commit, but superseded by separate branch bringing in default url